api/src/pcapi/connectors/api_allocine.py

```diff
@@ -51,6 +52,42 @@ def get_movie_list_page(after: str = "") -> allocine_serializers.AllocineMovieLi
     return validated_response
 
 
+def filter_invalid_edge(json_data: dict, locs: tuple) -> dict:
+    if not locs or set(locs[:2]) != {"movieShowtimeList", "edges"}:
+        # nothing to do (to complicated or locs is malformed), let it fail
+        return json_data
+
+    edge_index = locs[2]
+    invalid_edge = copy.deepcopy(json_data["movieShowtimeList"]["edges"][edge_index])
+
+    enumerate_edges = enumerate(json_data["movieShowtimeList"]["edges"])
+    json_data["movieShowtimeList"]["edges"] = [edge for idx, edge in enumerate_edges if idx != edge_index]
+
+    logger.error(
+        "One allocine show time edge removed (invalid)",
+        extra={"location": locs, "edge": invalid_edge},
+    )
+
+    return json_data
+
+
+def _parse_movie_showtimes(
+    json_data: dict, try_count: int = 0
+) -> allocine_serializers.AllocineMovieShowtimeListResponse:
+    try:
+        return allocine_serializers.AllocineMovieShowtimeListResponse.model_validate(json_data)
+    except pydantic.ValidationError as exc:
+        if try_count > 1:
+            # try two times, in case some errors were hidden
+            raise AllocineException(f"Error validating Allocine response. Error: {str(exc)}")
```

Comment:
Un simple raise ici est suffisant. Le formatting de l'erreur est fait dans la fonction appelante 
```suggestion
            # try two times, in case some errors were hidden
            raise
```